pkg/apis/openebs.io/v1alpha1/cas_pool.go

```diff
@@ -33,4 +56,25 @@ type CasPool struct {
 	// Namespace can be passed via storagepoolclaim as labels to decide on the
 	// execution of namespaced resources with respect to storagepool
 	Namespace string
+
+	// DiskList is the list of disks over which a storagepool will be provisioned
+	DiskList []string
+
+	// PoolType is the type of pool to be provisioned e.g. striped or mirrored
+	PoolType string
+
+	// MaxPool is the maximum number of pool that should be provisioned
+	MaxPools int
+
+	// MinPool is the minimum number of pool that should be provisioned
+	MinPools int
+
+	// Type is the CasPool type e.g. sparse or openebs-cstor
+	Type string
+
+	// reSync will decide whether the event is a reconciliation event
+	ReSync bool
+
+	// SparePoolCount is the number of pools that will be tried for creation as a part of reconciliation.
+	SparePoolCount int
```

Comment:
Change this to PendingPoolCount. Likewise, replace in other relevant places as well.